dev/integration_tests/ui/lib/license_check.dart

```diff
@@ -14,18 +14,18 @@ void main() {
 }
 
 class ShowLicenses extends StatelessWidget {
-  const ShowLicenses({Key key}) : super(key: key);
+  const ShowLicenses({Key? key}) : super(key: key);
 
   Widget _buildTestResultWidget(
     BuildContext context,
     AsyncSnapshot<List<LicenseEntry>> snapshot,
   ) {
-    final List<LicenseEntry> entries = snapshot.data;
+    final List<LicenseEntry>? entries = snapshot.data;
     String flutterPackage = '';
     final List<String> flutterParagraphs = <String>[];
     String enginePackage = '';
     final List<String> engineParagraphs = <String>[];
-    for (final LicenseEntry entry in entries) {
+    for (final LicenseEntry entry in entries!) {
```

Comment:
I don't like the `entries!` here, but I'm not sure what the right way to fail would be (I'm not familiar with this test).  @gspencergoog would you mind taking over this review?

Reply:
Sure, no problem.

Reply:
If you change the `entries` above to be non-nullable, this no longer needs the `!`.